src/main/kotlin/app/revanced/patches/tiktok/misc/login/disablerequirement/DisableLoginRequirementPatch.kt

```diff
@@ -1,24 +1,24 @@
-package app.revanced.patches.tiktok.misc.login.disablerequirement.patch
+package app.revanced.patches.tiktok.misc.login.disablerequirement
 
-import app.revanced.patcher.annotation.Description
-import app.revanced.patcher.annotation.Name
 import app.revanced.patcher.data.BytecodeContext
 import app.revanced.patcher.extensions.InstructionExtensions.addInstructions
 import app.revanced.patcher.patch.BytecodePatch
-import app.revanced.patcher.patch.annotations.Patch
-import app.revanced.patches.tiktok.misc.login.disablerequirement.annotations.DisableLoginRequirementCompatibility
+import app.revanced.patcher.patch.annotation.CompatiblePackage
+import app.revanced.patcher.patch.annotation.Patch
 import app.revanced.patches.tiktok.misc.login.disablerequirement.fingerprints.MandatoryLoginServiceFingerprint
 import app.revanced.patches.tiktok.misc.login.disablerequirement.fingerprints.MandatoryLoginServiceFingerprint2
 
-@Patch
-@Name("Disable login requirement")
-@Description("Do not force login.")
-@DisableLoginRequirementCompatibility
-class DisableLoginRequirementPatch : BytecodePatch(
-    listOf(
-        MandatoryLoginServiceFingerprint,
-        MandatoryLoginServiceFingerprint2
-    )
+@Patch(
+    name = "Disable login requirement",
+    description = "Disables the requirement to login to use the app.",
+    compatiblePackages = [
+        CompatiblePackage("com.ss.android.ugc.trill"),
+        CompatiblePackage("com.zhiliaoapp.musically")
+    ]
+)
```

Comment:
❓ : Description is not required
> This patch's description is similar to their name. Providing description might not be helpful when the patch's name is self-explanatory.

```suggestion
@Patch(
    name = "Disable login requirement",
    compatiblePackages = [
        CompatiblePackage("com.ss.android.ugc.trill"),
        CompatiblePackage("com.zhiliaoapp.musically")
    ]
)
```